extensions/pi-authentik/LLM_ENDPOINT_SETUP.md

```
@@ -0,0 +1,80 @@
+
+Examples:
+
+- `"modelFilters": ["*"]`
+- `"modelFilters": ["gpt-*"]`
+- `"modelFilters": ["gpt-*", "o3-*"]`
+
+If filters match nothing, the current implementation falls back to returning all discovered models.
+
+## Troubleshooting
+
+### "LLM base URL must end with /v1"
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)